README.md

```diff
@@ -72,160 +63,6 @@ implement support for each backend.
 
 ## OpenCensus Agent
 
-### <a name="agent-architecture-overview"></a>Architecture Overview
-
-On a typical VM/container, there are user applications running in some processes/pods with
-OpenCensus Library (Library). Previously, Library did all the recording, collecting, sampling and
-aggregation on spans/stats/metrics, and exported them to other persistent storage backends via the
-Library exporters, or displayed them on local zpages. This pattern has several drawbacks, for
-example:
-
-1. For each OpenCensus Library, exporters/zpages need to be re-implemented in native languages.
-2. In some programming languages (e.g Ruby, PHP), it is difficult to do the stats aggregation in
-process.
-3. To enable exporting OpenCensus spans/stats/metrics, application users need to manually add
-library exporters and redeploy their binaries. This is especially difficult when there’s already
-an incident and users want to use OpenCensus to investigate what’s going on right away.
-4. Application users need to take the responsibility in configuring and initializing exporters.
-This is error-prone (e.g they may not set up the correct credentials\monitored resources), and
-users may be reluctant to “pollute” their code with OpenCensus.
-
-To resolve the issues above, we are introducing OpenCensus Agent (Agent). Agent runs as a daemon
-in the VM/container and can be deployed independent of Library. Once Agent is deployed and
-running, it should be able to retrieve spans/stats/metrics from Library, export them to other
-backends. We MAY also give Agent the ability to push configurations (e.g sampling probability) to
-Library. For those languages that cannot do stats aggregation in process, they should also be
-able to send raw measurements and have Agent do the aggregation.
-
-For developers/maintainers of other libraries: Agent can also be extended to accept spans/stats/metrics from
-other tracing/monitoring libraries, such as Zipkin, Prometheus, etc. This is done by adding specific
-interceptors. See [Interceptors](#interceptors) for details.
-
-![agent-architecture](https://user-images.githubusercontent.com/10536136/46707027-dae59700-cbec-11e8-80ae-f16a8bdbacf4.png)
```

Comment:
Please keep the detailed architecture diagram on the README. Without it people will get lost on what it does, even with the originally bare minimum one, it confused the Halloween out of me before and I've seen it also confuse other people on this project.
The confusion lasted until the detailed image was put up that I figured out what interceptors did and then finally started implementing them. The word descriptions on the README themselves don't convey the gist of this project.

Reply:
OK that makes sense. I put back the high level workflow graph.